app/Http/Client/Opensea/OpenseaPendingRequest.php

```diff
@@ -101,7 +96,7 @@ public function getCollectionFloorPrice(string $collectionSlug): ?Web3Collection
         try {
             $response = $this->makeCollectionStatsRequest($collectionSlug);
 
-            $floorPrice = $response->json('total.floor_price');
+            $floorPrice = $response['total']['floor_price'] ?? null;
```

Comment:
will this throw if `total` does not exist in the response?

Reply:
nah

<img width="387" alt="Screenshot 2024-02-08 at 20 11 12" src="https://github.com/ArdentHQ/dashbrd/assets/6536260/0075d0c5-367b-42dd-9f06-c255317cf792">